packages/rum-core/src/domain/rumEventsCollection/action/actionCollection.ts

```diff
@@ -101,7 +101,7 @@ function processAction(
     customerContext,
     rawRumEvent: actionEvent,
     startTime: action.startClocks.relative,
-    domainContext: isAutoAction(action) ? { event: action.event } : {},
+    domainContext: isAutoAction(action) ? { events: action.events } : {},
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏we should keep the behavior of the deprecated field to not break customer current usages